StripePaymentSheet/StripePaymentSheet/Source/Internal/API Bindings/v1-elements-sessions/STPElementsSession.swift

```diff
@@ -211,6 +211,10 @@ extension STPElementsSession {
         return customer?.customerSession.mobilePaymentElementComponent.features?.paymentMethodRemoveLast ?? true
     }
 
+    var paymentMethodSetAsDefaultForPaymentSheet: Bool {
```

Comment:
Doesn't need to be in this PR, but we should probably rename xForPaymentSheet to xForMPE